content/docs/custom-plugins/configuring/index.md

```diff
@@ -0,0 +1,77 @@
+
+## Prerequisites
+
+Custom plugins need to be enabled for your Roadie instance. You can enable this by contacting us.
+
+## Configuring a plugin for your Roadie instance
+
+### Step 1. Create plugin definition in Roadie
+
+You can navigate to https://your-company.roadie.so/administration/custom-plugins where you will find a collection of cards detailing custom plugins configured for the Roadie instance. 
+
+![custom_plugins_page](custom_plugins_page.png)
+
+### Form
+
+You can create a new custom plugin definition by clicking "Add New Plugin" floating button. 
+![custom_plugin_form](custom_plugin_form.png)
+
+The values that need to be added are as follows:
+
+##### Package name
+Package field corresponds to the name of the NPM package. This can be a public NPM package hosted by Microsoft in npmjs.com or a private package securely hosted in Roadie Artifactory. If the package is hosted in Roadie Artifactory, the name of the package needs to start with `@<your-company>-roadie/`.
+
+
+##### Plugin Name
+The name of the plugin. This field can be arbitrarily chosen and will be used as a identifier when adding components from the plugin to Roadie as Cards, Tabs or Pages.
+
+
+##### Plugin version
+Semantic versioning definition for the plugin to use. This can be defined with standard NPM semantic versioning range identifiers like `^` or `~`, as well as keywords like `latest`.
+
+
+##### Components
+List of components that the plugin exposes. These are defined as a type and name pair. The type is  one of Card, Content or Page. The name needs to match the component name in code that is built into the plugin package.
+
+
+### Step 2. Installing
+
+Custom plugins are installed within the application from an external workflow. This can be triggered either by calling a webhook from your CI pipeline or manually or automatically when a new NPM package is published to private, secure Roadie Artifactory. Note that this installation step may take up to 20 minutes at the moment, though we are tirelessly working on bringing that number down.
+
+
+## Additional information
+
+### Plugin Statuses
+Each plugin exposes a status field indicating if they are available to be used within the Roadie instance. The statuses are as follows:
+
+#### Added
+Newly added plugin configuration. An attempt will be made to install this plugin during the next application installation cycle.
+
+##### Updated
+Updated plugin configuration. An older version of the plugin is still available to be used in the application and updates to version or components will be added during the next application installation cycle
+
+##### Installed successfully
+Application installation cycle has started and has successfully updated the application code. The plugin and components will be available to use when the next version of the application is rolled out.
```

Comment:
What is the "application" in this case? The plugin? Or the Backstage application? What is a roll out and how do I know when that will happen?